holmes/core/feedback.py

```diff
@@ -0,0 +1,175 @@
+from typing import Callable, Optional
```

Comment:
Instead of extending the holmes/interactive.py which is already pretty heavy, I used a feedback.py to simply the test.
We can consider moving more functionalities out of interactive.py